nodes/imgproc/histograms/cv_compareHist.py

```diff
@@ -0,0 +1,45 @@
+import uuid
+
+import bpy
+import cv2
+from ocvl.core.node_base import OCVLNodeBase
+
+from core.node_base import update_node
```

Comment:
Move this import to line up.
Something like this: `from ocvl.core.node_base import OCVLNodeBase, update_node `